### DISCUSSION TITLE
Learning sources

### DISCUSSION THREAD

**Discussion by DayvidAstua** [mebs/CM4-NAS#20] [General]
Hi I love your project, it is really inspiring to start making PCB's. So I would like to know some of your sources for learning and from other people who are experienced in PCB design. I don't know anything except some basic components like fuses, resistors, capacitors, etc. and the basics of electricity like voltage, current, resistance and its relation with each other.

**Comment by mebs** [comment]
Absolutely! I mentioned this subject here and there but it's a good idea to gather that info here.

I think the most important resource to get started is a good KiCad tutorial. There are many but I really enjoyed this one:

https://github.com/MalphasWats/hawk

This is exactly what I needed to get started with KiCad.
From there, it gets very specific and there's no single source that holds everything. Methodology is very important. As I was modifying the official IO board, a good way to make sure I didn't break anything was to research every component I was removing and try to understand what they do. Sometimes it can seem obvious, but it's also a good idea to keep both the CM4 and IO board datasheets open and double checking everything. You probably should also read through both datasheets once and research every word you don't know.

You don't know what you don't know so never assume anything. Make sure to stick to the CM4's specs, and it should be fine 😛
